RELEASES.md

```diff
@@ -0,0 +1,57 @@
+
+## Release branches and tags ##
+
+ScubaGear major and minor releases are built directly from the main branch.  Branch protections prevent direct push to the main branch.  All changes require a pull request and associated review prior to merge. 
+When a new release is planned, the latest commit to be included is tagged with its release versions (e.g., vX.Y.Z).  Patch versions are created from a separate release branch named `release-X.Y.Z` and are branched from the latest release tag or previous patch release branch which they are patching. The patch release branch contains only the additional changes for the patch along with version bumps.
+
+## Preparing ScubaGear release candidate ##
+
+The checklist below is used by the development team when it prepares a new release.  The goal of the list below is to ensure consistency and quality in the resulting releases.
+
+- [ ] Ensure all [blocked](https://github.com/cisagov/ScubaGear/labels/) issues and pull requests are resolved.
+- [ ] (future) Update CHANGELOG
+- [ ] Validate that all tests pass on CI for the release branch before proceeding
+- [ ] Update ModuleVersion in the [manifest](https://github.com/cisagov/ScubaGear/blob/main/PowerShell/ScubaGear/ScubaGear.psd1) to match release version
+- [ ] Update the module version in the [README.md](https://github.com/cisagov/ScubaGear/blob/main/README.md) badge image link and release download artifact name.
+- [ ] Check and update copyright dates in [manifest](https://github.com/cisagov/ScubaGear/blob/main/PowerShell/ScubaGear/ScubaGear.psd1) as needed
+- [ ] If baselines changed, update `baseline_version` in [Orchestrator module](https://github.com/cisagov/ScubaGear/blob/main/PowerShell/ScubaGear/Modules/Orchestrator.psm1)
+- [ ] Update and redact the sample report using the redaction tool and manual review
+- [ ] Check README for any necessary changes and documentation updates as needed
+- [ ] Build initial release candidate by manually triggering `Build and Sign Release` action with expected release name (vX.X.X) and release version (X.X.X) based on semantic versioning
+- [ ] Conduct automated release testing of each baseline
+- [ ] Fix critical defects deemed release blocking
+- [ ] Document non-critical issues for future development cycle
+- [ ] If fixes applied, restart release process
+
+## Publishing ScubaGear release candidate ##
+
+After running the `Build and Sign Release` workflow, a draft release will be visible to development team members for review and revision.  The checklist below is designed to ensure consistency in review and publishing of the release candidate as the final release. 
+
+- [ ] Update release notes to match expected format, including major new features, bug fixes, documentation improvements, and baseline updates Release notes should link to associated closed pull requests.
```

Comment:
This can be a separate issue. In case you haven't seen generated release notes can be configured with a [release.yml](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes#configuring-automatically-generated-release-notes) file to organize PRs with specfic labels, ie `enhancement`/`bug`/`documentation`/`baseline-document` under their respective headers `Major Changes`, `Bugs Fixed`, `Documentation`, and `Baselines`. Less work to do if we're having to group PRs manually.